Tasks/Common/codecoverage-tools/cobertura/cobertura.gradle.ccenabler.ts

```diff
@@ -0,0 +1,63 @@
+/// <reference path="../../../../definitions/Q.d.ts" />
+/// <reference path="../../../../definitions/string.d.ts" />
+/// <reference path="../../../../definitions/vsts-task-lib.d.ts" />
+/// <reference path="../../../../definitions/node.d.ts" />
+
+import * as util from "../utilities";
+import * as tl from "vsts-task-lib/task";
+import * as ccc from "../codecoverageconstants";
+import * as cc from "../codecoverageenabler";
+import * as str from "string";
+import * as os from "os";
+import * as Q from "q";
+
+export class CoberturaGradleCodeCoverageEnabler extends cc.CoberturaCodeCoverageEnabler {
+    // -----------------------------------------------------
+    // Enable code coverage for Cobertura Gradle Builds
+    // - enableCodeCoverage: CodeCoverageProperties  - ccProps
+    // -----------------------------------------------------    
+    public enableCodeCoverage(ccProps: { [name: string]: string }): Q.Promise<boolean> {
+        let _this = this;
+
+        _this.buildFile = ccProps["buildfile"];
+        let classFilter = ccProps["classfilter"];
+        let isMultiModule = ccProps["ismultimodule"] && ccProps["ismultimodule"] === "true";
+        let classFileDirs = ccProps["classfilesdirectories"];
+        let reportDir = ccProps["reportdirectory"];
+        let codeCoveragePluginData = null;
+
+        let filter = _this.extractFilters(classFilter);
+        let cobExclude = _this.applyFilterPattern(filter.excludeFilter);
+        let cobInclude = _this.applyFilterPattern(filter.includeFilter);
+
+        if (isMultiModule) {
+            codeCoveragePluginData = ccc.coberturaGradleMultiModuleEnable(cobExclude.join(","), cobInclude.join(","), classFileDirs, null, reportDir);
+        } else {
+            codeCoveragePluginData = ccc.coberturaGradleSingleModuleEnable(cobExclude.join(","), cobInclude.join(","), classFileDirs, null, reportDir);
+        }
+
+        try {
+            console.log("Code Coverage data will be appeneded to build file: " + _this.buildFile);
+            util.insertTextToFileSync(_this.buildFile, ccc.coberturaGradleBuildScript, codeCoveragePluginData);
```

Comment:
If i recall correctly, this is introduced in latest Gradle version, apparently old version of gradle build we don't have this. correct me